_events/sr2022/competition.md

```diff
@@ -31,12 +31,19 @@ We have a limited amount of free parking on site for competitors.
 
 ![parking information]({{ '/resources/sr2022/parking-map.png' | prepend: site.baseurl }})
 
+## Media Consent
+
+Before gaining access to the venue, everyone will need to sign a [media consent form][mcf-link]. Those under the age of 16 will need to get a parent or guardian to sign on their behalf.
+
+Please bring the signed form with you to get through reception faster. Media consent forms will also be available at reception.
+
```

Comment:
Suggest that we put this above the location information so that it's more prominent. The maps occupy quite a lot of the page.